html/opencraft_team_roles_2.html

```
@@ -189,25 +189,79 @@ <h3>Other Roles</h3>
-  </li>
-  <li>
-    If it's a non-urgent hosting issue, the owner of the current "Hosted Sites Maintenance" epic should
-    reply, and schedule any resulting work for an upcoming sprint.
+    If the email is related to a specific project/epic with a different epic owner, that epic owner can
+    reply directly (make sure the client owner and "OpenCraft &lt;help@opencraft.com&gt;" are CC'd).
   </li>
   <li>
-      If it's a non-urgent, non-hosting issue, the owner of the "Small Projects" epic should
-      reply, and schedule any resulting work for an upcoming sprint.
+    If the client owner is away, their designated backup person (usually the reviewer on the
+    "Support" epic/ticket for that client) should take over this role. (Or the firefighers
+    if no backup person was planned.)
   </li>
+</ul>
+<p>
+  We have a few clients with monthly development budgets. For those clients, the assigned client owner
+  is also responsible for reviewing the monthly budget before and during each sprint planning meeting,
+  to ensure that we are not going too far above or below the budget. They should refer to the
+  <a href="https://tasks.opencraft.com/rest/scriptrunner/latest/custom/showOpenCraftReport">Sprint Planning Report</a>
+  which can monitor and project the status of each budget as of the end of the upcoming sprint.
+</p>
+<p>
+  When we get a new client, or when the client owner changes (or we start working with a new
+  person coordinating things on the client's side, we send them this message:
 </p>
+<blockquote>
+  <p>
+    Welcome to OpenCraft : )
+  </p>
+  <p>
+    We are excited to work with you.
+  </p>
+  <p>
+    Here's a reference for how you can contact OpenCraft. Please keep this email available
+    somewhere so you can refer to it as needed.
+  </p>
+  <p>
+    <strong>Purr Sonnsname &lt;purr@opencraft.com&gt;</strong> will be your main technical
```

Do we have a process in place for if/when this "Purr Sonnsname" leaves? Clients might contact only them out of habit, or reply to an old mail. 

Maybe such mails should bounce with a helpful message or be redirected to help@opencraft.com?

---

Well, before that person leaves, their Support epics should be re-assigned, and anytime an epic is re-assigned, this email should be sent out. CC @gabrieldamours 

Does that make sense?

---

My only concern there is I don't know what happens when you remove the purr@opencraft.com account. If the email bounces / goes to help that's probably OK. If the email silently goes to an inbox that will never be checked then that could potentially cause an issue.

---

Once the Google Account would be deactivated during offboarding, it should bounce. That's not something that we always do quickly though, although that got better since @bradenmacdonald took over offboarding from me, which I was always deprioritizing :)